sal_interfaces/SALGenerics.xml

```diff
@@ -182,16 +119,34 @@
         <Count>1</Count>
     </item>
 </SALCommand>
+<SALCommand>
+    <Subsystem>SALGeneric</Subsystem>
+    <EFDB_Topic>SALGeneric_command_setAccess</EFDB_Topic>
```

Comment:
Please consider renaming the command to `setAuthList` or renaming the `authList` event to `access`, so it is more obvious that the command and event go together.